test/validate_test.js

```diff
@@ -79,6 +79,32 @@ describe("Validation", () => {
       });
     });
 
+    describe("data-url validation", () => {
+      let errors;
+
+      const schema = {
+        type: "object",
+        required: ["dataUrlWithName", "dataUrlWithoutName"],
+        properties: {
+          dataUrlWithName: { type: "string", format: "data-url" },
+          dataUrlWithoutName: { type: "string", format: "data-url" },
+        },
+      };
+
+      beforeEach(() => {
+        const formData = {
+          dataUrlWithName: "data:text/plain;name=file1.txt;base64,x=",
+          dataUrlWithoutName: "data:text/plain;base64,x=",
+        };
+        const result = validateFormData(formData, schema);
+        errors = result.errors;
+      });
+
+      it("Both should be fine", () => {
```

Comment:
Thanks for the test -- don't mean to be nitpicky, but it would really be great and cleaner if you could have separate `it` functions for the dataUrlWithName and a test case for dataUrlWithoutName.